build.gradle

```diff
@@ -177,6 +177,8 @@ subprojects {
                 }
             } else if (details.requested.group == 'log4j' && details.requested.name == 'log4j') {
                 details.useTarget group: 'org.apache.logging.log4j', name: 'log4j-1.2-api', version: '2.17.1'
+            } else if (details.requested.group == 'org.xerial.snappy' && details.requested.name == 'snappy-java') {
+                details.useTarget group: 'org.xerial.snappy', name: 'snappy-java', version: '1.1.10.1'
```

Comment:
Is this going to override the versions defined in individual packages like this?

https://github.com/opensearch-project/data-prepper/blob/4967df2eec0454cf7363bbde40e7d1dba4749436/data-prepper-plugins/s3-source/build.gradle#L33